CharlieBackend.AdminPanel/Services/MentorService.cs

```diff
@@ -57,9 +58,14 @@ public async Task<IList<MentorViewModel>> GetAllMentorsAsync(string accessToken)
 
         public async Task<MentorEditViewModel> GetMentorByIdAsync(long id, string accessToken)
         {
-            var mentor = await
-                            _apiUtil.GetAsync<MentorEditViewModel>($"{_config.Value.Urls.Api.Https}/api/mentors/{id}", accessToken);
-    
+            var mentorTask = _apiUtil.GetAsync<MentorEditViewModel>($"{_config.Value.Urls.Api.Https}/api/mentors/{id}", accessToken);
+            var coursesTask = _apiUtil.GetAsync<IList<CourseViewModel>>($"{_config.Value.Urls.Api.Https}/api/courses", accessToken);
+            var studentGroupTask = _apiUtil.GetAsync<IList<StudentGroupViewModel>>($"{_config.Value.Urls.Api.Https}/api/student_groups", accessToken);
```

Comment:
Why are temporary variables needed? Do inline instead.

Reply:
Everything is correct, since this is an async API request